Flaky internal test: e2e domain passes [firefox]

### Current behavior:

This test fails in our internal test sometimes. 

**CircleCI:** https://circleci.com/gh/cypress-io/cypress/330251
**Test Code:** https://github.com/cypress-io/cypress/blob/develop/packages/server/test/e2e/4_domain_spec.coffee#L16:L16

<img width="898" alt="Screen Shot 2020-05-19 at 2 20 21 PM" src="https://user-images.githubusercontent.com/1271364/82299831-05f4b700-99dc-11ea-8c00-0b197107830a.png">

```
AssertionError: expected exit code: expected 1 to equal 0
    at /root/cypress/packages/server/test/support/helpers/e2e.js:613:25
    at maybeVerifyExitCode (test/support/helpers/e2e.js:370:10)
    at exit (test/support/helpers/e2e.js:612:7)
    at copyDirItems (/root/cypress/node_modules/fs-extra/lib/copy/copy.js:151:21)
    at /root/cypress/node_modules/fs-extra/lib/copy/copy.js:145:12
    at go$readdir$cb (/root/cypress/node_modules/graceful-fs/graceful-fs.js:195:14)
    at FSReqCallback.oncomplete (fs.js:146:23)
From previous event:
    at Object.exec (test/support/helpers/e2e.js:721:6)
    at execFn (test/support/helpers/e2e.js:341:20)
    at Object.onRun (test/support/helpers/e2e.js:305:14)
    at Context.<anonymous> (test/support/helpers/e2e.js:344:22)
```

Looks like it failed to record the video?

```
Warning: We failed to record the video.

This error will not alter the exit code.

Error: ffmpeg exited with code 1: pipe:0: Function not implemented

    at ChildProcess.<anonymous> (/root/cypress/node_modules/fluent-ffmpeg/lib/processor.js:9:1)
    at ChildProcess.emit (events.js:9:1)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:9:1)

Error: ffmpeg exited with code 1: pipe:0: Function not implemented

    at ChildProcess.<anonymous> (/root/cypress/node_modules/fluent-ffmpeg/lib/processor.js:9:1)
    at ChildProcess.emit (events.js:9:1)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:9:1)

Error: ffmpeg exited with code 1: pipe:0: Function not implemented

    at ChildProcess.<anonymous> (/root/cypress/node_modules/fluent-ffmpeg/lib/processor.js:9:1)
    at ChildProcess.emit (events.js:9:1)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:9:1)
```

### Versions

4.5.0

--- a/packages/server/test/e2e/4_domain_spec.coffee
+++ b/packages/server/test/e2e/4_domain_spec.coffee
@@ -16,6 +16,7 @@ describe "e2e domain", ->
   e2e.it "passes", {
     spec: "domain*"
     snapshot: true
+    video: false
     config: {
       hosts
     }

